From a6e6495fc1369e1c9f8db872411eea7e0a4f2a67 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Tue, 9 Jul 2019 12:57:07 +0100
Subject: [PATCH 1/7] spec: Update seavgabios dependency

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <1562677027-9619-1-git-send-email-mrezanin@redhat.com>
Patchwork-id: 89436
O-Subject: [RHEL-AV-8.1.0 qemu-kvm PATCH] spec: Update seavgabios dependency
Bugzilla: 1725664
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Danilo de Paula <ddepaula@redhat.com>
RH-Acked-by: Wainer dos Santos Moschetta <wainersm@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

Branch: rhel-av-8.1.0/master-4.0.0
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1725664
Brew: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=22574916
Upstream: n/a

We introduced new vga bios files (vgabios-ramfb.bin and
vgabios-bochs-display.bin) recently. To prevent dangling symlinks,
we bump required seavgabios-bin package to version adding these
bios files.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 redhat/qemu-kvm.spec.template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qemu-kvm.spec b/qemu-kvm.spec
index e6c0980..a43ebc8 100644
--- a/qemu-kvm.spec
+++ b/qemu-kvm.spec
@@ -244,7 +244,7 @@ Requires: edk2-aarch64
 %endif
 
 %ifnarch aarch64 s390x
-Requires: seavgabios-bin >= 1.10.2-1
+Requires: seavgabios-bin >= 1.12.0-3
 Requires: ipxe-roms-qemu >= 20170123-1
 %endif
 %ifarch %{power64}
-- 
1.8.3.1

